samples/apps/logging/logging.cpp

```diff
@@ -298,6 +298,13 @@ namespace loggingapp
         // SNIPPET: public_table_access
         auto records_handle = ctx.tx.template rw<RecordsMap>(PUBLIC_RECORDS);
         records_handle->put(params["id"], in.msg);
+        // SNIPPET_START: set_claims_digest
+        if (in.record_claim)
+        {
+          ctx.rpc_ctx->set_claims_digest(
+            std::move(ccf::ClaimsDigest::Digest::from_string(in.msg)));
```

Comment:
Is the `std::move()` required here since the result of `from_string()` will be a rvalue?